builder/scripts.d/50-x265.sh

```diff
@@ -20,6 +20,11 @@ ffbuild_dockerbuild() {
         -DENABLE_CLI=OFF
         -DCMAKE_ASM_NASM_FLAGS=-w-macro-params-legacy
         -DENABLE_ALPHA=ON
+        -DENABLE_NEON=ON
+        -DENABLE_NEON_DOTPROD=OFF
```

Comment:
These are common flags which will disable dotprod and i8mm on macOS builds too. We need to move those to the `[[ $TARGET != mac* ]]` sections below

Reply:
oops i forgot macos builder also shared this